<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettingsLikeFifteen</class>
 <widget class="QWidget" name="FormSettingsLikeFifteen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBoxGameType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Swap Mode</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icons/swap</normaloff>:/icons/swap</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slide Mode</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/icons/slide</normaloff>:/icons/slide</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSpecifiedPosition">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Swap target position</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButtonBlankRandom">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonBlankSpecified">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Specified</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cells</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxSplitY">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="hSliderSplitY">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBoxSplitX">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="hSliderSplitX">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>hSliderSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSplitX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormSettingsLikeFifteen</receiver>
   <slot>udpateGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSplitY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormSettingsLikeFifteen</receiver>
   <slot>udpateGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSliderSplitX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSliderSplitY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>udpateGrid()</slot>
 </slots>
</ui>
